runtime/JavaScript/src/antlr4/error/Errors.js

```diff
@@ -78,7 +78,7 @@ LexerNoViableAltException.prototype.constructor = LexerNoViableAltException;
 LexerNoViableAltException.prototype.toString = function() {
     var symbol = "";
     if (this.startIndex >= 0 && this.startIndex < this.input.size) {
-        symbol = this.input.getText((this.startIndex,this.startIndex));
+        symbol = this.input.getText(this.startIndex);
```

Comment:
should this not be: get Text(new Interval(...

Reply:
Could you explain why it should be `getText(new Interval(...))`? I'm having trouble following the code but I think it should be `getText(this.startIndex, this.startIndex)`.

The only two places that define a `getText` method that I could find are `InputStream` and `BufferedTokenStream`. Of those two the former takes a start and end as separate parameters, and the latter takes an interval.

This function we're modifying is on `LexerNoViableAltException` and one of those is only constructed from `LexerATNSimulator`. In this class every function received an `input` parameter and they seem to be `InputStream`s as later on the `getText` version with two arguments is used (unless that call is wrong). I'm thinking it makes sense as well that you can't have a token stream yet as the lexer is in the process of making one and is if this exception is thrown then it failed.